src/redux/actions/types.js

```diff
@@ -0,0 +1,9 @@
+export const resetpassword = {
+  RESET_PASSWORD_SUCCESS: 'RESET_PASSWORD_SUCCESS',
+  RESET_PASSWORD_FAILURE: 'RESET_PASSWORD_FAILURE'
+};
+export const URLs = {
+  BACK_END1: 'https://inviticus-staging.herokuapp.com/',
+  BACK_END: 'http://localhost:8000/',
```

Comment:
Please remove this link for localhost if we not going to use it remotely. Thanks